Neos.Flow/Classes/Http/Component/TrustedProxiesComponent.php

```diff
@@ -54,10 +54,13 @@ public function injectSettings(array $settings)
         if ($this->settings['proxies'] === ['*']) {
             $this->settings['proxies'] = '*';
         }
+        if (is_string($this->settings['proxies'])) {
+            $this->settings['proxies'] = array_map('trim', explode(',', $this->settings['proxies']));
+        }
         if (!is_array($this->settings['proxies']) && $this->settings['proxies'] !== '*') {
```

Comment:
I guess this check is now even obsolete, since it will not be a single string at this point any more.

Reply:
It can stilll be `'*'` (now)…